controllers/generators/images.go

```diff
@@ -0,0 +1,73 @@
+package generators
+
+import (
+	"encoding/json"
+	"io/ioutil"
+
+	aimlv1beta1 "github.com/pachyderm/openshift-operator/api/v1beta1"
+)
+
+// ImageCatalog consists the certified images
+// required to deploy a Pachyderm cluster
+type ImageCatalog struct {
+	Pachd    *aimlv1beta1.ImageOverride `json:"pachd,omitempty"`
+	Dash     *aimlv1beta1.ImageOverride `json:"dash,omitempty"`
```

Comment:
Out of curiosity, where is dash used? I don't think we have a dash pod anymore, but I'm actually not sure.